third_party/move/move-model/src/well_known.rs

```diff
@@ -45,15 +45,21 @@ pub const TYPE_NAME_GET_SPEC: &str = "type_name::$get";
 /// The well-known name of the first parameter of a method.
 pub const RECEIVER_PARAM_NAME: &str = "self";
 
-/// The well-known abort codes used by the compiler. These are marked
-/// by lowest 6 bytes of a sha256 of the string "Move 2 Abort Code",
-/// appended with two bytes for the error type.
-/// TODO: add a check at runtime that user is not clashing with reserved
-/// codes?
-pub const WELL_KNOWN_ABORT_CODE_BASE: u64 = 0xD8CA26CBD9BE << 16;
+/// The well-known abort codes used by the compiler. Those confirm
+/// to the error category standard as defined in `errors.move`
```

Comment:
Can you please point to that file?  (Is that third_party/move/move-stdlib/nursery/sources/errors.move?)  If you can point to the doc explaining this, that might also be useful.  This comment isn't quite self-explanatory to me.

Then again, I may just be tired.  Others can approve if they like.

Reply:
Added, its `move-stdlib/sources/error.move`. There was a spelling error

Reply:
It's not useful to resolve a conversation when you've just answered a question.  It means I have to dig around to find out if you answered me.